ykrt/src/location.rs

```diff
@@ -104,6 +104,12 @@ impl Location {
         }
     }
 
+    pub(super) unsafe fn from_locationinner(ls: LocationInner) -> Self {
```

Comment:
`from_location_inner` looks a bit nicer to me.

Reply:
It's because `locationinner` is corresponding to the `LocationInner` type. I don't feel strongly about this one.